<launch>
	<arg name="enable_octomap" value="true"/>
	<arg name="enable_goto_controller" value="false"/>
    
	<group if="$(arg enable_octomap)">
    	<node pkg="octomap_server" type="octomap_server_node" name="octomap_server" args=" " cwd="node" output="screen">
    		<param name="frame_id"            	  value="/odom" />
    		<remap from="/cloud_in" to="/camera/depth/points" /> <!-- simulator -->
    		<!--remap from="/cloud_in" to="/camera/depth_registered/points" /--> <!-- real turtlebot -->
		</node>
    </group>
    
    <group if="$(arg enable_goto_controller)">
    	<node pkg="control_turtlebot" type="controller_turtlebot.py" name="controller_turtlebot" respawn="true" output="screen" />
    </group>

</launch>
